packages/core/src/components/HierarchyPicker/Search/index.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import Fuse from 'fuse.js';
+import Autocomplete from '../../Autocomplete';
+import SearchResult from './SearchResult';
+import fuseLoader from './fuseLoader';
+import {
+  Formatter,
+  ItemPickedHandler,
+  ItemShape,
+  SearchItemShape,
+  SearchItemResult,
+  TreePath,
+  TopicOriginKey,
+} from '../types';
+
+export type Props = {
+  items?: ItemShape[];
+  onItemPicked: ItemPickedHandler;
+  formatter: Formatter;
+  query?: string;
+  noResultsLabel: NonNullable<React.ReactNode>;
+  placeholder?: string;
+  indexParentPath?: boolean;
+  fuseOptions?: Fuse.FuseOptions<any>;
+  width: number;
+  maxHeight?: number;
+};
+
+const defaultFuseOptions: Fuse.FuseOptions<any> = {
```

Comment:
Avoid using `any`. We should type the structure correctly.

Reply:
didn't add this but will update